state-chain/pallets/cf-vaults/src/migrations/v4.rs

```diff
@@ -13,11 +13,9 @@ pub struct Migration<T, I>(PhantomData<(T, I)>);
 
 impl<T: crate::Config<I>, I: 'static> OnRuntimeUpgrade for Migration<T, I> {
 	fn on_runtime_upgrade() -> frame_support::weights::Weight {
-		if <T::Chain as Chain>::NAME == "Arbitrum" {
-			ChainInitialized::<T, I>::put(false);
-		} else {
-			ChainInitialized::<T, I>::put(true)
-		}
```

Comment:
Wont this migration run for arbitrum's instance of vault pallet?

Reply:
The problem is that there are 2 migrations, v3 and v4. v3 should *not* run so we also need to disable v4 for arbitrum. But this is ok: the default value is false anyway. So we just need to insert true for the existing vaults.

Reply:
ok, but shouldn't we then remove the assert_ne!() because it will run v4 migration and will panic?

Reply:
I removed the Arbitrum vault from `PalletMigrations` and I added the assert to prevent someone adding it again, because adding it causes the v3 migration to fail. The runtime upgrade works, at least on my machine :) Feel free to try it out.